src/components/Tooltip/styled.tsx

```diff
@@ -35,6 +35,11 @@ export const Styles = createGlobalStyle<{ variant: Variant }>`
 
       .tippy-content {
         padding: 0;
+        z-index: 0;
```

Comment:
Let's use the values from `theme.honeycomb.zIndex` - if there is no relevant one, feel free to create a new one.